docs/themes/oso-tailwind/layouts/partials/header.html

```diff
@@ -2,12 +2,16 @@
     class="static pin-t lg:sticky lg:z-50  top-0 pin-t h-8 w-full max-w-8xl mx-auto bg-primary flex flex-none items-center">
     <div id="header-page-title" class="flex-grow">
         <p class="text-white font-base text-sm  text-center truncate">
-            Want to get started? Jump into <a class="underline" href="https://join-slack.osohq.com">our Slack</a> and an engineer from the core team can help you out.
+            {{ if eq $.Page.Language.Lang "python" }}
+                Using SQLAlchemy? Check out <a class="underline" href="{{ ref . "/new-roles" }}">early access</a> to the next version of our SQLAlchemy integration.
+            {{ else }}
+                Want to get started? Jump into <a class="underline" href="https://join-slack.osohq.com">our Slack</a> and an engineer from the core team can help you out.
+            {{ end }}
         </p>
     </div>
 </nav>
 <nav id="header"
-    class="static lg:sticky top-8 pin-t h-16 lg:h-20 lg:z-50 max-w-8xl mx-auto bg-primary-dark flex flex-none items-center">
+    class="static 2xl:sticky top-8 h-16 lg:h-20 lg:z-50 max-w-8xl mx-auto bg-primary-dark flex flex-none items-center">
```

Comment:
@samscott89 removed the floating header so that there is more space to read. (It was getting pretty tight on a 13'' laptop)